src/components/register/register.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { IRegisterFormProperties } from '../../@types/types';
+import './register.scss';
+
+export function Register(): ReactElement {
+
+  const history = useHistory();
+
+  const [form, setState] = useState<IRegisterFormProperties>({
+    username: '',
+    password: '',
+    email: '',
+  });
+
+  function redirectToLogin() {
+    history.push('/login');
+  }
+
+  const updateField = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setState({
+      ...form,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  return (
+    <div className="c-register">
+      <h2 className="c-register__header">Register</h2>
+      <form className="c-register__form">
+        <input type="text" placeholder="Username" name="username" value={form.username}
+               onChange={updateField}/>
+        <input type="text" placeholder="Email" name="email" value={form.email} onChange={updateField}/>
+        <input type="password" placeholder="Password" name="password" value={form.password}
+               onChange={updateField}/>
+        <button type="button">
+          Create account
+        </button>
+        <p className="c-register__login">Or
+          <span className="c-register__login--link" onClick={redirectToLogin}> login</span>
```

Comment:
Would it be possible to make it an actual link? `<a href="">`. It would be better for accessibility purposes and is in fact a redirecting reference.

Reply:
@paroxp But it will reload page. Do we want to do that?

Reply:
Or should i use something like: 
```onClick={(event) => {
    event.preventDefault();
    history.push('/register');
}} (or create function)?

Reply:
So, yeah. We want to prevent the default behaviour of the link in case JS is enabled, but in the same time, we should not rely fully on JS if we can replicate the same behaviour with a tag. So, yeah, your code (probably as a separate function if you're planning on re-using this pattern, would be good.

Perhaps, this calls for our custom `Link` component?